GVFS/GVFS.Common/Git/DiffTreeResult.cs

```diff
@@ -167,6 +171,23 @@ public static DiffTreeResult ParseFromLsTreeLine(string line, string repoRoot)
             }
         }
 
+        /// <summary>

```

Comment:
I don't think this comment block is adding any value. It's just duplicating exactly what the names of the method and params already tell me.